spec-main/api-web-contents-spec.ts

```diff
@@ -1191,6 +1195,23 @@ describe('webContents module', () => {
       expect(currentRenderViewDeletedEmitted).to.be.false('current-render-view-deleted was emitted');
     });
 
+    it('does not destroy child window when wrong url and nativeWindowOpen set to true', async () => {
```

Comment:
Wouldn't it be better to test directly the (not) emission of the relevant event ("current-render-view-deleted") instead of one consequence of that event? This is what the surrounding tests are doing.